xpartamupp/xpartamupp.py

```diff
@@ -188,9 +223,6 @@ def _muc_online(self, presence):
         if nick == self.nick:
             return
 
-        if jid.resource not in ['0ad', 'CC']:
-            return
-
```

Comment:
These if-statements prevented users not connected with 0ad, but with another XMPP client, from getting 0ad-related IQ-stanzas from the bots. While this is no issue for the users, as their clients shouldn't do anything with the received IQ-stanza, it'll add more load to the bots as they'll have to send and receive more messages.